src/components/com_tjcertificate/administrator/sql/install.mysql.utf8.sql

```diff
@@ -26,6 +26,8 @@ CREATE TABLE IF NOT EXISTS `#__tj_certificate_issue` (
   `generated_body` text NOT NULL,
   `client` varchar(100) NOT NULL,
   `client_id` int(11) NOT NULL,
+  `client_issued_to` int(11) NOT NULL,
+  `client_issued_to_name` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
```

Comment:
In the demo, the datatype was `Text`, and here its `Varchar`. Please make it `varchar(400)`